docker/tensorflow-aarch64/README.md

```diff
@@ -91,7 +91,7 @@ For example:
 To choose between the different tensorflow versions use '--tf_version 1' for TensorFlow 1.15.2 or '--tf_version 2' for TensorFlow 2.3.0. The default value is set to tf_version=2.
 For the base build: This will generate an image named 'DockerTest/ubuntu/base-v$tf_version', hyphenated with the version of TensorFlow chosen.
 
-TensorFlow can optionally be built with oneDNN, using the '--onednn' or '--dnnl' flag. Tensorflow 1.x is built with oneDNN 0.21.3 (MKL-DNN) and Tensorflow 2.x is built with oneDNN 1.5.1.
+TensorFlow can optionally be built with oneDNN, using the '--onednn' or '--dnnl' flag. Tensorflow 1.x is built with oneDNN 0.21.3 (MKL-DNN) and Tensorflow 2.x is built with oneDNN 1.7-rc.
```

Comment:
I think this should be `1.7` rather than `1.7-rc` as v1.7 is released now (and the Dockerfile appears to be pulling the 1.7 tag).